config/chime_science_run_gpu.yaml

```diff
@@ -103,9 +103,9 @@ updatable_config:
   frb_gain:
     kotekan_update_endpoint: json
     frb_gain_dir: /mnt/frb-archiver/daily_gain_solutions/Latest_FRB
-  pulsar_gain:
+  tracking_gain:
```

Comment:
This should have a parallel PR in `coco` to update the names of these endpoints.  And the Pulsar group will also need to update their scripts.

Reply:
I've created https://github.com/chime-experiment/coco/pull/216 for this change in `coco`

Reply:
And the Pulsar script is changed in CHIME-Pulsar-Timing/ch_psr_gains#7.